uom_security/static/description/index.html

```diff
@@ -0,0 +1,1136 @@
+    <div class="oe_row">
+        <h3 class="oe_slogan">
+            Product Template Form (Warehouse Menu)
+        </h3>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Original</h4>
+            <img class="oe_picture oe_screenshot" src="product.template.form1.warehouse.src.png">
+        </div>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Modificated</h4>
+            <img class="oe_picture oe_screenshot" src="product.template.form1.warehouse.new.png">
+        </div>
+    </div>
+
+    <!--
+
+    Product Template Search (Manufacturing Menu)
+
+    .. model: product.template
+
+    .. status: DONE (warehouse)
+    .. view: product.product_template_search_view
+        <filter string="Default Unit of Measure" icon="terp-mrp" domain="[]" context="{'group_by' : 'uom_id'}"/>
+    -->
+
+    <div class="oe_row">
+        <h3 class="oe_slogan">
+            Product Template Search (Manufacturing Menu)
+        </h3>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Original</h4>
+            <img class="oe_picture oe_screenshot" src="product.template.search.warehouse.src.png">
+        </div>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Modificated</h4>
+            <img class="oe_picture oe_screenshot" src="product.template.search.warehouse.new.png">
+        </div>
+    </div>
+
+    <!--
+
+    HR Expense Form
+
+    .. model: hr.expense.expense
+
+    .. status: DONE (warehouse)
+    .. TODO evaluate the current security of this view.
+    .. view: hr_expense.view_expenses_form
+        <field name="line_ids"> ...  <form> ... <field name="uom_id"/>
+        <field name="line_ids"> ...  <tree> ... <field name="uom_id"/>
+    -->
+
+    <div class="oe_row">
+        <h3 class="oe_slogan">
+            Expense Form (Human Resources Menu)
+        </h3>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Original</h4>
+            <img class="oe_picture oe_screenshot" src="hr.expense.expense.form.src.png">
+        </div>
+        <div class="oe_span6">
+            <h4 class="oe_slogan">Modificated</h4>
+            <img class="oe_picture oe_screenshot" src="hr.expense.expense.form.new.png">
+        </div>
+    </div>
+
+
+</section>
+
+<section class="oe_container">
+    <h2 class="oe_slogan">General Accountng Operations</h2>
+
+    <!--
+
+    Analytic Journal Entry Form
+
+    .. model: account.analytic.line
+    .. original: account.analytic.line.form.src.png
+    .. TODO check with nhomar about this field.
+
+    .. status: DONE (purchase)
+    .. view account.view_account_analytic_line_form
+        <field name="product_uom_id" on_change="on_change_unit_amount(product_id, unit_amount, company_id, product_uom_id)" class="oe_inline"/>
+
+    .. view account.account_analytic_line_extended_form
+        <field name="product_uom_id" on_change="on_change_unit_amount(product_id, unit_amount, company_id, product_uom_id)" class="oe_inline"/>
+
+    -->
```

Comment:
thank you @nhomar. I will remove the unnecessary info and only have left the xml id of the view to make more easy the edition of this module in new iterations.

Regards